.travis.yml

```diff
@@ -2,20 +2,24 @@ language: generic
 os:
     - linux
     - osx
+compiler:
+    - gcc
```

Comment:
is specifying the compiler still necessary since we force compiling against gcc on OSX anyway?

Reply:
well, probably not needed indeed